### PR TITLE
--socket=fallback-x11, FEDC support

### PR DESCRIPTION
**Pull request by BigmenPixel0**
Used --socket=fallback-x11 instead of --socket=x11, added FEDC support.

### CODE REVIEW

**Comment by x80486** [comment]
...merge?

**Comment by BigmenPixel0** [comment]
> ...merge?

Maybe I should remove the commit with the new runtime so that you accept the previous pr?

**Comment by hfiguiere** [comment]
you need to rebase this I guess.

**Comment by hfiguiere** [comment]
bot, build

File: nl.hjdskes.gcolor3.json [hunk 1]
```diff
@@ -24,7 +24,13 @@
...
                     "type": "git",
                     "url": "https://github.com/flatpak/libportal.git",
-                    "commit": "bff3289781ef27388e75f772d5fe9dee20fdfb53"
+                    "commit": "bff3289781ef27388e75f772d5fe9dee20fdfb53",
+                    "tag": "0.4",
+                     "commit": "f68764e288ede516d902b131cc4fadded3804059",
+                     "x-checker-data": {
+                         "type": "git",
+                         "tag-pattern": "^([\\d.]+)$"
+                     }
```
Comment (hfiguiere) [inline nl.hjdskes.gcolor3.json:33]: for the record this was a bad idea. 0.5 is API incompatible. I'm reverting this.

Comment (BigmenPixel0) [reply nl.hjdskes.gcolor3.json:33]: You haven't returned yet. Do it to me?

Comment (hfiguiere) [reply nl.hjdskes.gcolor3.json:33]: my bad I thought i had pushed some changes and I didn't.

Will do.

Comment (hfiguiere) [reply nl.hjdskes.gcolor3.json:33]: this is done now.

Comment (hfiguiere) [reply nl.hjdskes.gcolor3.json:33]: via #13

**Pull request merged**